<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModelTree</class>
 <widget class="QWidget" name="ModelTree">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="minimumSize">
      <size>
       <width>326</width>
       <height>0</height>
      </size>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>322</width>
        <height>317</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>317</height>
       </size>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="4" column="0">
        <widget class="QLabel" name="label_4">
         <property name="minimumSize">
          <size>
           <width>141</width>
           <height>57</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Comic Sans MS</family>
           <pointsize>12</pointsize>
           <stylestrategy>PreferAntialias</stylestrategy>
           <kerning>false</kerning>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <property name="text">
          <string>Param 4</string>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignJustify|Qt::AlignVCenter</set>
         </property>
         <property name="margin">
          <number>21</number>
         </property>
         <property name="indent">
          <number>0</number>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::NoTextInteraction</set>
         </property>
         <property name="buddy">
          <cstring>doubleSpinBox_4</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_3">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>33</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Comic Sans MS</family>
           <pointsize>11</pointsize>
           <stylestrategy>PreferAntialias</stylestrategy>
           <kerning>false</kerning>
          </font>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="accelerated">
          <bool>true</bool>
         </property>
         <property name="decimals">
          <number>4</number>
         </property>
         <property name="minimum">
          <double>-99999.990000000005239</double>
         </property>
         <property name="maximum">
          <double>99999.990000000005239</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_1">
         <property name="minimumSize">
          <size>
           <width>37</width>
           <height>57</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Comic Sans MS</family>
           <pointsize>12</pointsize>
           <stylestrategy>PreferAntialias</stylestrategy>
           <kerning>false</kerning>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <property name="text">
          <string>Param 1</string>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignJustify|Qt::AlignVCenter</set>
         </property>
         <property name="margin">
          <number>21</number>
         </property>
         <property name="indent">
          <number>0</number>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::NoTextInteraction</set>
         </property>
         <property name="buddy">
          <cstring>doubleSpinBox</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="3">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_4">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>33</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Comic Sans MS</family>
           <pointsize>11</pointsize>
           <stylestrategy>PreferAntialias</stylestrategy>
           <kerning>false</kerning>
          </font>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="accelerated">
          <bool>true</bool>
         </property>
         <property name="decimals">
          <number>4</number>
         </property>
         <property name="minimum">
          <double>-99999.990000000005239</double>
         </property>
         <property name="maximum">
          <double>99999.990000000005239</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_3">
         <property name="minimumSize">
          <size>
           <width>141</width>
           <height>57</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Comic Sans MS</family>
           <pointsize>12</pointsize>
           <stylestrategy>PreferAntialias</stylestrategy>
           <kerning>false</kerning>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <property name="text">
          <string>Param 3</string>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignJustify|Qt::AlignVCenter</set>
         </property>
         <property name="margin">
          <number>21</number>
         </property>
         <property name="indent">
          <number>0</number>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::NoTextInteraction</set>
         </property>
         <property name="buddy">
          <cstring>doubleSpinBox_3</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QDoubleSpinBox" name="doubleSpinBox">
         <property name="minimumSize">
          <size>
           <width>67</width>
           <height>33</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Comic Sans MS</family>
           <pointsize>11</pointsize>
           <stylestrategy>PreferAntialias</stylestrategy>
           <kerning>false</kerning>
          </font>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="accelerated">
          <bool>true</bool>
         </property>
         <property name="decimals">
          <number>4</number>
         </property>
         <property name="minimum">
          <double>-99999.990000000005239</double>
         </property>
         <property name="maximum">
          <double>99999.990000000005239</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="minimumSize">
          <size>
           <width>141</width>
           <height>57</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Comic Sans MS</family>
           <pointsize>12</pointsize>
           <stylestrategy>PreferAntialias</stylestrategy>
           <kerning>false</kerning>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <property name="text">
          <string>Param 2</string>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignJustify|Qt::AlignVCenter</set>
         </property>
         <property name="margin">
          <number>21</number>
         </property>
         <property name="indent">
          <number>0</number>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::NoTextInteraction</set>
         </property>
         <property name="buddy">
          <cstring>doubleSpinBox_2</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>33</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Comic Sans MS</family>
           <pointsize>11</pointsize>
           <stylestrategy>PreferAntialias</stylestrategy>
           <kerning>false</kerning>
          </font>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="accelerated">
          <bool>true</bool>
         </property>
         <property name="decimals">
          <number>4</number>
         </property>
         <property name="minimum">
          <double>-99999.990000000005239</double>
         </property>
         <property name="maximum">
          <double>99999.990000000005239</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSlider" name="sliderIncrement1">
         <property name="minimumSize">
          <size>
           <width>54</width>
           <height>0</height>
          </size>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
         <property name="singleStep">
          <number>10</number>
         </property>
         <property name="pageStep">
          <number>100</number>
         </property>
         <property name="value">
          <number>200</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="invertedAppearance">
          <bool>true</bool>
         </property>
         <property name="invertedControls">
          <bool>true</bool>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksBelow</enum>
         </property>
         <property name="tickInterval">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSlider" name="sliderIncrement2">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
         <property name="singleStep">
          <number>10</number>
         </property>
         <property name="pageStep">
          <number>100</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="invertedAppearance">
          <bool>true</bool>
         </property>
         <property name="invertedControls">
          <bool>true</bool>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksBelow</enum>
         </property>
         <property name="tickInterval">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSlider" name="sliderIncrement3">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
         <property name="singleStep">
          <number>10</number>
         </property>
         <property name="pageStep">
          <number>100</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="invertedAppearance">
          <bool>true</bool>
         </property>
         <property name="invertedControls">
          <bool>true</bool>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksBelow</enum>
         </property>
         <property name="tickInterval">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QSlider" name="sliderIncrement4">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
         <property name="singleStep">
          <number>10</number>
         </property>
         <property name="pageStep">
          <number>100</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="invertedAppearance">
          <bool>true</bool>
         </property>
         <property name="invertedControls">
          <bool>true</bool>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksBelow</enum>
         </property>
         <property name="tickInterval">
          <number>100</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QTreeWidget" name="treeWidget">
     <property name="minimumSize">
      <size>
       <width>230</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="lineWidth">
      <number>3</number>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectItems</enum>
     </property>
     <property name="indentation">
      <number>13</number>
     </property>
     <property name="uniformRowHeights">
      <bool>true</bool>
     </property>
     <property name="animated">
      <bool>true</bool>
     </property>
     <property name="headerHidden">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string>1</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
